src/components/Card.js

```diff
@@ -1,4 +1,5 @@
 import Popup from "./Popup.js";
+import Api from "./API.js";
 
```

Comment:
When I open the confirmation modal there are errors 

![image](https://github.com/Gaktines/se_project_aroundtheus/assets/54933269/6331e487-8d18-4ad5-8df4-185ee08bd9e9)

Reply:
fixed.